feature/fonts

NataliYurchenko opened the pull request:
add font link to index.html;  marked fonts.scss;  import  font file to main.scss; 
***As for me, lines 14+15 are needlessin in file variables.scss;

NataliYurchenko committed:
add fonts

add font link into index.html;

igorklyopov commented on src/sass/base/_fonts.scss:
```diff
@@ -1 +1,10 @@
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
```
line-height лучше указывать в относительных единицах(%). 16px - это 133%

NataliYurchenko replied:
line-height поставила у %.

igorklyopov commented on src/sass/base/_fonts.scss:
```diff
@@ -1 +1,10 @@
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  //   letter-spacing: ;
```
Упоминаний о letter-spacing в макете вроде нет. Можешь оставить пока, но потом только не забудь удалить. Или можно сразу удалять ненужные комментарии. Если, что - данное свойство можно указать точечно.

NataliYurchenko replied:
видалила  letter-spacing

igorklyopov commented on src/sass/base/_fonts.scss:
```diff
@@ -1 +1,10 @@
+body {
+  color: #000000;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
```
используем переменные

NataliYurchenko replied:
юзала змінні

igorklyopov commented on src/index.html:
```diff
@@ -128,6 +128,12 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
```
шрифты лучше подключать локально (нужно скачать и подключить). Основные преимущества: Меньше запросов на сервер. Можно дополнительно оптимизировать (в данном случае - это необязательно, можешь не заморачиваться если, что. Но желательно добавить хотя бы font-display: swap)

NataliYurchenko replied:
намагалась - зверни особливу увагу. Досі юзала через фонт лінк в хеді.

igorklyopov reviewed (commented):
> add font link to index.html; marked fonts.scss; import font file to main.scss;
> ***As for me, lines 14+15 are needlessin in file variables.scss;

lines 14 + 15 are not needed in variables.scss file - remove them

NataliYurchenko committed:
feature/fonts

remove lines 14 + 15 in variables.scss file;
line-height поставила у %;
видалила  letter-spacing;
проставила змінну в font body;
видалила лінк шрифтів з html;
намагалася підключити шрифт локально(не впевнена, що все вийшло як ти цього хотів. - ОБОВ'ЯЗКОВО зверни увагу).
видалила з common.scss інфо про font-family(код закоментила - видали,якщо він і справді не потрібен).

igorklyopov committed:
Merge branch 'main' into feature/fonts
